pipelines/terraform/cluster/main.tf

```diff
@@ -0,0 +1,99 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-2"
+
+  assume_role {
+    role_arn = "${var.aws_account_role_arn}"
+  }
+}
+
+data "aws_caller_identity" "current" {}
+
+module "gsp-persistent" {
+  source       = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/gsp-persistent"
+  cluster_name = "${module.gsp-network.cluster-name}"
+  dns_zone     = "${var.dns_zone}"
+}
+
+module "gsp-network" {
+  source       = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/gsp-network"
+  cluster_name = "${var.cluster_name}"
+}
+
+module "gsp-cluster" {
+  source       = "git::https://github.com/alphagov/gsp-terraform-ignition//modules/gsp-cluster"
```

Comment:
Is it worth adding a `module_ref` variable to make it easier to create a cluster testing a branch of `gsp-terraform-ignition`?

Reply:
Good idea ...

Reply:
added, but not tested